### PR TITLE
test(function-tree): ensure inputs are passed correctly with empty paths

### PR DESCRIPTION
**Pull request by christianalfoni**
Just a test to ensure that an edge case in Cerebral 1 does not happen in Cerebral 2

### CODE REVIEW

File: packages/function-tree/src/FunctionTree.test.js [hunk 1]
```diff
@@ -240,4 +240,25 @@ describe('FunctionTree', () => {
     assert(branchStartCount, 2)
     assert(branchEndCount, 2)
   })
+  it('should pass correct input on empty paths', () => {
+    let branchStartCount = 0
+    let branchEndCount = 0
```
Comment (henri-hulski) [inline packages/function-tree/src/FunctionTree.test.js]: Lint errors: branchStartCount and branchEndCount are assigned a value but never used.

Comment (christianalfoni) [inline packages/function-tree/src/FunctionTree.test.js]: DOH! Will rebase soon ;-)
